CMakeLists.txt

```diff
@@ -186,6 +187,24 @@ else()
   message(STATUS "NiftyPET projector support disabled.")
 endif()
 
+if(NOT DISABLE_STIR_CUDA)
+  ## check 
+  find_package(CUDAToolkit REQUIRED)
+  find_package(CUDA REQUIRED)
```

Comment:
are you sure this is ok? I think this is outdated and `check_language` should be used